util/migration-rocksdb/src/lib.rs

```diff
@@ -217,7 +217,7 @@ impl Manager {
 	}
 
 	/// Adds new migration rules.
-	pub fn add_migration<T>(&mut self, migration: T) -> io::Result<()> where T: Migration {
+	pub fn add_migration<T: 'static>(&mut self, migration: T) -> io::Result<()> where T: Migration {
```

Comment:
this is needed because `migrations` are stored as trait objects `Vec<Box<dyn Migration>>` where `Box<dyn Trait>` is sugar for `Box<dyn Trait + 'static`.